docs/src/main/asciidoc/spanner.adoc

```diff
@@ -317,8 +317,11 @@ CREATE TABLE items (
 ) PRIMARY KEY (grandParentId, parentId, id)
 ----
 
-Note that embedded properties' column names must all be unique.
+Note that the following restrictions apply when you use embedded objects:
 
+- Embedded properties' column names must all be unique.
+- Embedded properties must not be passed through a constructor and the property must be mutable; otherwise you'll get an error, such as `SpannerDataException: Column not found: parent`.
```

Comment:
```suggestion
- Embedded properties must not be passed through a constructor and the property must be mutable; otherwise you'll get an error, such as `SpannerDataException: Column not found`.
```